CHANGELOG.md

```diff
@@ -21,6 +22,62 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v0.6.1
+
+This is a patch release that predominantly includes updated performance tests
```

Comment:
```suggestion
This is a patch release that predominantly includes updated conformance tests
```

Reply:
Good catch thank you :) a83679beeb56d7789dedd373fb23605e46b3b6c9